webapp/src/components/login-modal.vue

```diff
@@ -45,17 +45,17 @@
         <b-form-input 
           v-model="signInCreds.password" 
           type="password" 
-          :state="signInValidationResults[1]"
+          :state="signInValidationResults.password"
           class="custom-dialog-input"
           placeholder="Enter password"
           id="password"
           @update="helper.password = true"
         />
         <b-form-text v-show="showPasswordHelper" class="text-center">
-          Between 8 and 18 characters including uppercase, numeric and special characters
+          Can be any non-zero number of characters including uppercase, numeric and special characters
```

Comment:
Have we changed password-strength rules or if this is just for the login? If this is just for the login, then I would suggest not writing all this, just saying "Enter your password" is enough. The rules only apply when signing up.